mod/search/start.php

```diff
@@ -317,7 +317,12 @@ function search_remove_ignored_words($query, $format = 'array') {
 	$query = stripslashes(strip_tags($query));
 	
 	$words = explode(' ', $query);
```

Comment:
We really should use preg_split here instead. We can use a pattern and specify not returning empty items. Using explode plus a filter is confusing to read.